app/components/edit/EditServices.jsx

```diff
@@ -8,8 +8,9 @@ const EditServices = ({
 }) => (
   <li className="edit--section--list--item">
     <ul className="edit--section--list--item--sublist edit--service--list">
-      {
-        services.map((service, index) => (
+      {services
+        .sort((a, b) => a.name - b.name)
```

Comment:
You probably don't want to subtract strings, since this mostly likely won't do what want it to do. The subtraction operator in JavaScript is only defined for numbers, so it'll attempt to cast both sides to a number before performing the subtraction operation.

Here's some examples:

```js
> "abc" - "bde"
NaN
> "3" - "2"
1
> "foo" - "42"
NaN
```

Yes, in the middle expression, those are both strings that contain only numbers and therefore get converted to actual numbers.

---

Since we use lodash, I'd recommend using the [`sortBy`](https://lodash.com/docs/#sortBy) function, which allows you to specify a property to sort by.